### ISSUE TITLE
Feature: Console commands

### ISSUE THREAD

**Issue by lukewatts** [Postleaf/postleaf#27]
It would be nice to have a feature similar to Laravel, Symfony, Doctrine, Grav CMS etc. where a user can run commands from the console to perform the following tasks:
## Init
`postleaf init`
Setup db and files/folders required before startup and perhaps walk a user through prompts to set the `.env` variables. This would also be a good way to auto generate the random hash, similar to how Laravel does it on install.
## Package manager
`postleaf theme <theme-name>`
Install themes e.g. `postleaf theme empower`

## Backup
`postleaf backup`
Backup/zip the following directories `cache`, `data`, `themes`, `uploads`

**Comment by claviska** [comment]
I agree. Init and backups are great examples. What other functions would be useful from the command line?

**Comment by lukewatts** [comment]
Backup/restore would be good. Clear cache also. And a search perhaps to show available themes. Also update to update postleaf

**Comment by claviska** [comment]
Updated the list. Eventually themes will be installed/removed directly from the admin panel. Any advantage to having it in the CLI?

**Comment by lukewatts** [comment]
Simpler to automate. Could install Postleaf with the preferred theme with a shell script/Docker/Vagrant/Chef etc

**Comment by juan-manuel-alberro** [comment]
What about a tool / command to migrate the posts from another CMS into **Postleaf**? Maybe only one tool (for the most used CMS, **Wordpress** for e.x.) as a base and then other contributors can create more tools based on that.
Now I'm using Hugo (static `.md` files) in my site and I'm trying to figure it out how can I migrate all posts into Postleaf 😀
